test/built-ins/Object/entries/exception-during-enumeration.js

```diff
@@ -2,8 +2,8 @@
 // This code is governed by the BSD license found in the LICENSE file.
 
 /*---
+id: sec-object.entries
```

Comment:
I noticed that, but I'm following the pattern used on the current spec, as:

- https://tc39.github.io/ecma262/#sec-object.assign
- https://tc39.github.io/ecma262/#sec-object.create
- https://tc39.github.io/ecma262/#sec-object.defineproperties
- ...

My suggestion is that we should keep it, I may open a PR on the proposal you linked **or** change the ids to `pending` until they are merged on the spec.